CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 CHANGELOG
 =========
 
+0.3.0 ()
+
+* Added support for new minFraud Insights outputs. These are:
+    * `/credit_card/brand`
+    * `/credit_card/type`
+    * `/device/id`
+    * `/email/is_free`
+    * `/emai/is_high_risk`
+* The `Warning.input` attribute has been replaced by `Warning.inputPointer`,
+  which is a JSON Pointer rather than array.
+* The `ScoreResponse.id` and `InsightsResponse.id` attributes are now `UUID`
```

Comment:
These are private. You should probably rephrase this in terms of the getter. Same with the warning attributes.